app/models/ability.rb

```diff
@@ -71,6 +75,10 @@ def initialize_wimi(user)
     can [:read, :create, :update], Contract, responsible_id: user.id
     can :ts_wimi_actions, TimeSheet, contract: { responsible_id: user.id }
     can :see_wimi_actions, TimeSheet, contract: { responsible_id: user.id }, handed_in: true
+    can :send_to_admin, TimeSheet, contract: { responsible_id: user.id }, status: 'accepted', signed: true, wimi_signed: true
+    cannot :send_to_admin, TimeSheet do |ts|
+      ts.has_been_sent_to_admin?
+    end
```

Comment:
Can the `cannot :send_to_admin` statement be moved to a shared location, to avoid code duplication?

Reply:
Stick it into `initialize_after`